modules/local/filter_orthogroups.nf

```diff
@@ -52,17 +54,21 @@ process FILTER_ORTHOGROUPS {
     msaDir=\$( cd ${orthofinder_outdir}/Orthogroup_Sequences/; pwd )
 
     # Create the column
-    echo "file" > colname.txt
-    tail -n+2 all_ogs_counts.csv | cut -f1 -d"," | sed "s|.*|\${msaDir}/&.fa|g" | sed '1 i\\file' > all_og_fpaths.txt
     tail -n+2 spptree_core_ogs_counts.csv | cut -f1 -d"," | sed "s|.*|\${msaDir}/&.fa|g" | sed '1 i\\file' > spptree_core_og_fpaths.txt
     tail -n+2 genetree_core_ogs_counts.csv | cut -f1 -d"," | sed "s|.*|\${msaDir}/&.fa|g" | sed '1 i\\file' > genetree_core_og_fpaths.txt
 
-    # Combine them
-    paste -d"," all_ogs_counts.csv all_og_fpaths.txt > tmp && mv tmp all_og_fpaths.csv
-    paste -d"," spptree_core_ogs_counts.csv spptree_core_og_fpaths.txt > tmp && mv tmp spptree_core_ogs_counts.csv
-    paste -d"," genetree_core_ogs_counts.csv genetree_core_og_fpaths.txt > tmp && mv tmp genetree_core_ogs_counts.csv
+    mkdir SpeciesTreeFastas
+    mkdir GeneTreeFastas
 
-    # Clean up
-    rm *og_fpaths.txt
+    while IFS= read -r line
+    do
+        # Copy the file to the destination directory
+        cp "\$line" SpeciesTreeFastas/
+    done < spptree_core_og_fpaths.txt
+    while IFS= read -r line
+    do
+        # Copy the file to the destination directory
+        cp "\$line" GeneTreeFastas/
+    done < genetree_core_og_fpaths.txt
```

Comment:
can this not be done with the publishDir directive?

Reply:
Nope, `publishDir` doesn't determine what gets uploaded, it determines where things get uploaded to